python/xorbits/_mars/services/cluster/api/web.py

```diff
@@ -193,6 +193,14 @@ async def fetch_node_log(self):
                     size, address=address, offset=offset
                 )
 
+    @web_api("workers", method=["post"], cache_blocking=True)
+    async def request_workers(self):
+        worker_num = int(self.get_argument(name="worker_num", default="0"))
```

Comment:
worker_num is the required argument. So when this parameter is missing, a default value should not be provided, but an exception should be thrown directly.